app/scripts/locales/base.json

```diff
@@ -443,6 +443,7 @@
     "setGenClearSeconds": "In {} seconds",
     "setGenClearMinute": "In a minute",
     "setGenMinInstead": "Minimize the app instead of close",
+    "setGenMinOnCopy": "Minimize on copy",
```

Comment:
Let's be more elaborate and mention that it's only password copy

Reply:
It minimizes on all copy though. use case: open app/site requiring password, hotkey to open keeweb, find record, copy user name, paste, tab, hotkey to open keeweb again, copy password, paste, enter.

Reply:
This is not how it should behave, please change it.

Reply:
Sorry I read it wrong. I mean, it should minimize on copying FIELDS, i.e. what you write, exactly. But not when you copy something in a text field.

Reply:
For example, if you copy password in a generator, it shouldn't minimize.

Reply:
ah, got it, makes sense, thanks, will make the change

Reply:
I think that's expected, it should copy from details, right?

Reply:
I guess this comment was to the previous thread, yes, it all makes sense, thank you